packages/coil-extension/scripts/reinstall-six.sh

```diff
@@ -11,16 +11,16 @@ set -ex
 # So can override with the beta
 SIX_PKG=${SIX_PKG:-com.sec.android.app.sbrowser}
 
-# Uinstall old extension
-adb uninstall com.coil.android.six
-
-# Force stop Samsung Internet
-adb shell am force-stop $SIX_PKG
-# Clear Samsung Internet shared settings
-adb shell pm clear $SIX_PKG
+## Uinstall old extension
```

Comment:
Unsure if it's just the APK beta, but it seems no longer need to force stop the browser, clear the preferences and restart. Simply updating the extension APK seems to be picked up.